consensus/safety-rules/src/test_utils.rs

```diff
@@ -187,3 +194,14 @@ pub fn test_storage(signer: &ValidatorSigner) -> PersistentSafetyStorage {
     let storage = Box::new(InMemoryStorage::new());
     PersistentSafetyStorage::initialize(storage, signer.private_key().clone(), waypoint)
 }
+
+pub fn rotate_consensus_key(
```

Comment:
if this isn't used in more than one place, let's put it in the same file where it is used. Or just do it inline since it is only used once.

Reply:
please address